xml/System/Exception.xml

```diff
@@ -1364,7 +1364,7 @@ Stack Trace:
       </ReturnValue>
       <Docs>
         <summary>Gets a string representation of the immediate frames on the call stack.</summary>
-        <value>A string that describes the immediate frames of the call stack.</value>
+        <value>A string that describes the immediate frames of the call stack. If no stack trace is available (such as prior to being thrown), this will be `null`.</value>
```

Comment:
Is "being thrown" correct here or it should it rather be 
```suggestion
        <value>A string that describes the immediate frames of the call stack. If no stack trace is available (such as prior to stack unwinding, this will be `null`.</value>
```
or "stack walking", etc. Or a combination of these?

Reply:
Maybe a combination? "…such as prior to stack unwinding from a `throw` statement"?

Reply:
Sounds good.

Can the stack be unwound w/o throwing too? (I don't know, but docs should be accurate)

Reply:
Made a new commit with the wording tweak.